Backend/ServiceLayer/Services/TokenService.cs

```diff
@@ -17,5 +18,13 @@ public string GenerateToken()
             string hex = BitConverter.ToString(b).Replace("-", "");
             return hex;
         }
+
+        public string GenerateSignature(string plaintext, Application app)
```

Comment:
Should this plaintext parameter be strongly typed? Like will signing always use the same values as part of the plaint text? Using just a string then anyone can set the plaintext as any combination of values. Just for thought, what you think